### PR TITLE
fix: update inputfield pattern and support translation in every notification

### PR DESCRIPTION
**Pull request by lizable**
After merging this PR, every notification will be translated to current language settings.
Also, there was a minor bug exporting csv menu in the users page (Admin Only), and It will be fixed via this PR.

### CODE REVIEW

File: src/components/backend-ai-console.ts [hunk 1]
```diff
@@ -612,6 +612,8 @@ export default class BackendAIConsole extends connect(store)(LitElement) {
    * Open the user preference dialog.
    */
   async _openUserPrefDialog() {
+    let fields = ['email', 'username', 'password', 'full_name'];
+    await globalThis.backendaiclient.user.get(this.user_id, fields);
```
Comment (inureyes) [inline src/components/backend-ai-console.ts]: @lizable what is the purpose of these two lines?

**Review by inureyes — CHANGES_REQUESTED** [review]
Remove mentioned lines if not needed. :)

File: src/components/backend-ai-console.ts (hunk 1 above)
Comment (lizable) [reply src/components/backend-ai-console.ts]: My mistakes. Probably didn't erased in the middle of conflict handling.
I removed it right away.

**Review by adrysn — APPROVED** [review]
LGTM.

**Pull request merged**